packages/compartment-mapper/src/import-hook.js

```diff
@@ -15,7 +15,8 @@
  *   ImportHook,
  *   ImportNowHook,
  *   RedirectStaticModuleInterface,
- *   StaticModuleType
+ *   StaticModuleType,
+ VirtualModuleSource
```

Comment:
this looks odd. Did the IDE add it for you? Is it supposed to be here?

Reply:
it sure did!